modules/partytown/vue/plugins/partytown.server.ts

```diff
@@ -1,17 +1,28 @@
 import type { App } from 'vue'
-import fs from 'fs'
+import fs from 'fs/promises'
 import { join } from 'node:path'
 import { libDirPath } from '@builder.io/partytown/utils'
+import usePartytownConfig from '#ioc/config/PARTYTOWN_CONFIG'
+import IS_DEVELOPMENT from '#ioc/config/IS_DEVELOPMENT'
+
+let partytownSnippet: string | undefined = undefined
 
 export const after = async (app: App, ctx?: any) => {
-  const partyTownSnippet = fs.readFileSync(join(libDirPath(), './partytown.js'), 'utf-8')
-  const partyTownConfig = `partytown = { debug: false, forward: [], lib: "'${libDirPath()}'" }`
+  const { debug, forward } = usePartytownConfig()
+  const libPath = IS_DEVELOPMENT ? libDirPath() + '/' : ''
+
+  if (!partytownSnippet) {
+    const data = await fs.readFile(join(libDirPath(), './partytown.js'), 'utf-8')
```

Comment:
No need for the data variable. Just use `partytownSnipper = await fs.readFile(...`